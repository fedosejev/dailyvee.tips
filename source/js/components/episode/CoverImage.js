import React from 'react';

const CoverImage = ({ videoUrl, coverImage, showCoverImage, number }) => {
  return (
    <a href={videoUrl} target="_blank">
      <img
        src={ showCoverImage ? `./images/${coverImage}` : `http://placehold.it/1280x720?text=DAILYVEE` }
        title={`DailyVee #${number}`}
        alt={`DailyVee #${number}`}
        className="episodeCoverImage"
      />
    </a>
  );
};

export default CoverImage;
