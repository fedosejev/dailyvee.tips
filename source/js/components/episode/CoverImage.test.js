import React from 'react';
import CoverImage from './CoverImage';
import renderer from 'react-test-renderer';

describe('CoverImage', () => {
  test('SHOULD render correctly', () => {
    const tree = renderer.create(
      <CoverImage />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
