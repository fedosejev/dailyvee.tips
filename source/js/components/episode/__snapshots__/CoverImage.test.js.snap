// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoverImage SHOULD render correctly 1`] = `
<a
  href={undefined}
  rel="noopener noreferrer"
  target="_blank"
>
  <img
    alt="DailyVee #undefined"
    className="episodeCoverImage"
    src="http://placehold.it/1280x720?text=DAILYVEE"
    title="DailyVee #undefined"
  />
</a>
`;
